--- pulseaudio-4.0/configure.ac.orig	2013-06-08 13:07:47.119786074 +0200
+++ pulseaudio-4.0/configure.ac	2013-06-08 13:09:53.796450100 +0200
@@ -991,6 +991,8 @@
 
 PA_MACHINE_ID="${sysconfdir}/machine-id"
 AX_DEFINE_DIR(PA_MACHINE_ID, PA_MACHINE_ID, [D-Bus machine-id file])
+PA_PA_MACHINE_ID="${sysconfdir}/pulse/machine-id"
+AX_DEFINE_DIR(PA_PA_MACHINE_ID, PA_PA_MACHINE_ID, [PA machine-id file])
 PA_MACHINE_ID_FALLBACK="${localstatedir}/lib/dbus/machine-id"
 AX_DEFINE_DIR(PA_MACHINE_ID_FALLBACK, PA_MACHINE_ID_FALLBACK,
 	      [Fallback machine-id file])
--- pulseaudio-8.0/src/pulsecore/core-util.c.orig	2016-01-22 19:14:53.600528288 +0100
+++ pulseaudio-8.0/src/pulsecore/core-util.c	2016-01-22 19:19:09.026774778 +0100
@@ -3066,6 +3066,20 @@
      * identifier that suits our needs perfectly. If it's not, we fall back to
      * the hostname, which is not as good, since it can change over time. */
 
+    /* First we try locally configured ID */
+
+    if ((f = pa_fopen_cloexec(PA_PA_MACHINE_ID, "r"))) {
+        char ln[34] = "", *r;
+
+        r = fgets(ln, sizeof(ln)-1, f);
+        fclose(f);
+
+        pa_strip_nl(ln);
+
+        if (r && ln[0])
+            return pa_utf8_filter(ln);
+    }
+
     /* We search for the machine-id file from four locations. The first two are
      * relative to the configured installation prefix, but if we're installed
      * under /usr/local, for example, it's likely that the machine-id won't be
