--- pulseaudio-15.0/meson.build.orig	2021-07-27 22:02:27.711868000 +0200
+++ pulseaudio-15.0/meson.build	2021-07-28 00:14:10.525724158 +0200
@@ -137,6 +137,7 @@
 cdata.set('PA_API_VERSION', pa_api_version)
 cdata.set('PA_PROTOCOL_VERSION', pa_protocol_version)
 cdata.set_quoted('PA_MACHINE_ID', join_paths(sysconfdir, 'machine-id'))
+cdata.set_quoted('PA_PA_MACHINE_ID', join_paths(sysconfdir, 'pulse', 'machine-id'))
 cdata.set_quoted('PA_MACHINE_ID_FALLBACK', join_paths(localstatedir, 'lib', 'dbus', 'machine-id'))
 cdata.set_quoted('PA_SRCDIR', join_paths(meson.current_source_dir(), 'src'))
 cdata.set_quoted('PA_BUILDDIR', meson.current_build_dir())
--- pulseaudio-8.0/src/pulsecore/core-util.c.orig	2016-01-22 19:14:53.600528288 +0100
+++ pulseaudio-8.0/src/pulsecore/core-util.c	2016-01-22 19:19:09.026774778 +0100
@@ -3066,6 +3066,20 @@
      * identifier that suits our needs perfectly. If it's not, we fall back to
      * the hostname, which is not as good, since it can change over time. */
 
+    /* First we try locally configured ID */
+
+    if ((f = pa_fopen_cloexec(PA_PA_MACHINE_ID, "r"))) {
+        char ln[34] = "", *r;
+
+        r = fgets(ln, sizeof(ln)-1, f);
+        fclose(f);
+
+        pa_strip_nl(ln);
+
+        if (r && ln[0])
+            return pa_utf8_filter(ln);
+    }
+
     /* We search for the machine-id file from four locations. The first two are
      * relative to the configured installation prefix, but if we're installed
      * under /usr/local, for example, it's likely that the machine-id won't be
