--- pulseaudio-1.0/configure.ac.orig	2011-09-27 09:55:58.000000000 +0200
+++ pulseaudio-1.0/configure.ac	2011-10-12 20:57:56.702352030 +0200
@@ -1112,6 +1112,9 @@
 AC_SUBST(pulseconfdir, ["${sysconfdir}/pulse"])
 AX_DEFINE_DIR(PA_DEFAULT_CONFIG_DIR, pulseconfdir, [Location of configuration files])
 
+PA_PA_MACHINE_ID="${sysconfdir}/pulse"
+AX_DEFINE_DIR(PA_PA_MACHINE_ID, PA_PA_MACHINE_ID, [PA machine-id file])
+
 AC_DEFINE_UNQUOTED(PA_BUILDDIR, "${ac_pwd}/src", [Location of uninstalled binaries])
 
 #### Mac OSX specific stuff #####
--- pulseaudio-0.9.21/src/pulsecore/core-util.c.orig	2009-11-23 04:57:07.000000000 +0100
+++ pulseaudio-0.9.21/src/pulsecore/core-util.c	2010-06-09 00:20:25.361930253 +0200
@@ -2560,6 +2560,20 @@
     /* The returned value is supposed be some kind of ascii identifier
      * that is unique and stable across reboots. */
 
+    /* First we try locally configured ID */
+
+    if ((f = fopen(PA_PA_MACHINE_ID, "r"))) {
+        char ln[34] = "", *r;
+
+        r = fgets(ln, sizeof(ln)-1, f);
+        fclose(f);
+
+        pa_strip_nl(ln);
+
+        if (r && ln[0])
+            return pa_utf8_filter(ln);
+    }
+
     /* First we try the D-Bus UUID, which is the best option we have,
      * since it fits perfectly our needs and is not as volatile as the
      * hostname which might be set from dhcp. */
