--- pulseaudio-4.0/configure.ac.orig	2013-06-08 13:07:47.119786074 +0200
+++ pulseaudio-4.0/configure.ac	2013-06-08 13:09:53.796450100 +0200
@@ -991,6 +991,8 @@
 
 PA_MACHINE_ID="${sysconfdir}/machine-id"
 AX_DEFINE_DIR(PA_MACHINE_ID, PA_MACHINE_ID, [D-Bus machine-id file])
+PA_PA_MACHINE_ID="${sysconfdir}/pulse/machine-id"
+AX_DEFINE_DIR(PA_PA_MACHINE_ID, PA_PA_MACHINE_ID, [PA machine-id file])
 PA_MACHINE_ID_FALLBACK="${localstatedir}/lib/dbus/machine-id"
 AX_DEFINE_DIR(PA_MACHINE_ID_FALLBACK, PA_MACHINE_ID_FALLBACK,
 	      [Fallback machine-id file])
--- pulseaudio-3.0/src/pulsecore/core-util.c.orig	2012-12-22 08:18:53.590608852 +0100
+++ pulseaudio-3.0/src/pulsecore/core-util.c	2012-12-22 09:05:12.940550922 +0100
@@ -2893,6 +2893,20 @@
     /* The returned value is supposed be some kind of ascii identifier
      * that is unique and stable across reboots. */
 
+    /* First we try locally configured ID */
+
+    if ((f = pa_fopen_cloexec(PA_PA_MACHINE_ID, "r"))) {
+        char ln[34] = "", *r;
+
+        r = fgets(ln, sizeof(ln)-1, f);
+        fclose(f);
+
+        pa_strip_nl(ln);
+
+        if (r && ln[0])
+            return pa_utf8_filter(ln);
+    }
+
     /* First we try the /etc/machine-id, which is the best option we
      * have, since it fits perfectly our needs and is not as volatile
      * as the hostname which might be set from dhcp. */
