--- pulseaudio-0.9.5/configure.ac~	2007-02-24 14:56:39.000000000 +0100
+++ pulseaudio-0.9.5/configure.ac	2007-02-24 14:57:01.000000000 +0100
@@ -253,10 +253,6 @@
 # Check for open64 to know if the current system does have open64() and similar functions
 AC_CHECK_FUNCS([open64])
 
-#### [lib]iconv ####
-
-AM_ICONV
-
 ###################################
 #      External libraries         #
 ###################################
